mycpp/leaky_mylib.cc

```diff
@@ -8,8 +8,18 @@ mylib::FormatStringer gBuf;
 
 namespace mylib {
 
-Str* StrFromBuf(const Buf& buf) {
-  return ::StrFromC(buf.data_, buf.len_);
+Str* StrFromBuf(const Buf* buf) {
+  return ::StrFromC(buf->data_, buf->len_);
```

Comment:
This makes a copy, but we can look at removing that in a follow-up

Reply:
Ack